### DISCUSSION TITLE
Lack of Resources in the TV Series Tab

### DISCUSSION THREAD

**Discussion by NinhoNO** [jellyfin/jellyfin-androidtv#3138] [General]
Hello everyone, how are you?

I would like to request a feature that already exists in the web and mobile versions but that I couldn't find in the TV app version. I often use these sub-tabs to, for example, "organize" my files by streaming services, as I believe it makes it much easier to find specific content.

I'm not sure if this option is available and I simply haven't found it, but I wanted to share my suggestion. Thank you!

![Jellyfin](https://github.com/jellyfin/jellyfin-androidtv/assets/148784964/d4b7cbcf-c5ec-4771-aa0d-4613da9b58cb)

**Comment by nielsvanvelzen** [comment]
The "TV Networks" tab is not currently available in the Android TV app. I guess it could be added to the smart screen.
